GitUI/Editor/BlameAuthorMargin.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+using GitExtUtils.GitUI;
+using ICSharpCode.TextEditor;
+using Microsoft.VisualStudio.Threading;
+
+namespace GitUI.Editor
+{
+    public class GitBlameDisplay
+    {
+        public JoinableTask<Image> Avatar { get; set; }
```

Comment:
❗️ No no no, we must not return tasks, especially from properties. It is semantically incorrect to "await a property".
If you want to defer you could return a `Lazy<Image>`, but even then I'm not sure it would be correct